COMDOX-1734: Updated SCP topics for clarity (#941)
commerce-docs/microsite-commerce-storefront: src/content/docs/get-started/architecture/commerce-services-and-backends.mdx

Commerce Services include <Term>Catalog Service</Term>, <Term>Live Search</Term>, Product Recommendations, and core GraphQL on your instance for carts, checkout, accounts, and other flows not covered by the hosted catalog layer.

> All three backend options—Commerce on Cloud or on-premises, Adobe Commerce as a Cloud Service, and Adobe Commerce Optimizer—connect to these services. To compare them, see [Backend options](/get-started/backends/).

Reviewer: ```suggestion
All backend options connect to these services. To compare them, see [Backend options](/get-started/backends/).
```

### Storefront Compatibility package

> The Storefront Compatibility package extends the Commerce GraphQL schema with the reads and writes that cart, checkout, account, and order drop-ins require. Without it on the Commerce deployment that serves those flows, the drop-ins cannot talk to your backend as they expect. See [Storefront Compatibility Package](/setup/configuration/storefront-compatibility/) for who installs it on each backend. On Adobe Commerce as a Cloud Service and Adobe Commerce Optimizer, Adobe manages this package for you as part of the fully managed model in [Backend options](/get-started/backends/). On Commerce on Cloud or on-premises (PaaS) with Adobe Commerce Optimizer, install it with Composer on your Commerce server before you finish storefront settings in your GitHub project.

Reviewer: ```suggestion
The Storefront Compatibility package extends the default Commerce GraphQL schema with additional support that drop-ins require. Without it on the Commerce deployment that serves those flows, the drop-ins cannot talk to your backend as they expect. See [Storefront Compatibility Package](/setup/configuration/storefront-compatibility/) for who installs it on each backend. On Adobe Commerce as a Cloud Service and Adobe Commerce Optimizer, Adobe manages this package for you as part of the fully managed model in [Backend options](/get-started/backends/). On Commerce on Cloud or on-premises (PaaS) with Adobe Commerce Optimizer, install it with Composer on your Commerce server before you finish storefront settings in your GitHub project.
```

### Storefront Compatibility package

> The Storefront Compatibility package extends the Commerce GraphQL schema with the reads and writes that cart, checkout, account, and order drop-ins require. Without it on the Commerce deployment that serves those flows, the drop-ins cannot talk to your backend as they expect. See [Storefront Compatibility Package](/setup/configuration/storefront-compatibility/) for who installs it on each backend. On Adobe Commerce as a Cloud Service and Adobe Commerce Optimizer, Adobe manages this package for you as part of the fully managed model in [Backend options](/get-started/backends/). On Commerce PaaS or on-premises with Adobe Commerce Optimizer, install it with Composer on your Commerce server before you finish storefront settings in your GitHub project.

Reviewer: ```suggestion
The Storefront Compatibility package extends the Commerce GraphQL schema with the reads and writes that cart, checkout, account, and order drop-ins require. Without it on the Commerce deployment that serves those flows, the drop-ins cannot talk to your backend as they expect. See [Storefront Compatibility Package](/setup/configuration/storefront-compatibility/) for who installs it on each backend. On Adobe Commerce as a Cloud Service and Adobe Commerce Optimizer, Adobe manages this package for you as part of the fully managed model in [Backend options](/get-started/backends/). On Commerce on Cloud or on-premises (PaaS) with Adobe Commerce Optimizer, install it with Composer on your Commerce server before you finish storefront settings in your GitHub project.
```

| Product Recommendations | Optional (recommended). Adobe Sensei units in the Commerce Admin. It needs storefront event collection when you do not use the classic PHP theme. | <Link href="https://experienceleague.adobe.com/en/docs/commerce/product-recommendations/getting-started/headless" text="Integration without classic theme" /> <br/> <Link href="/merchants/content-customizations/product-recommendations/" text="Drop-in setup" /> |
| Core GraphQL | Required. GraphQL API on your Commerce instance for cart, checkout, account, orders, and other flows; hosted catalog services do not replace it. | <Link href="https://developer.adobe.com/commerce/webapi/graphql/" text="GraphQL overview" /> |
| Data Connection | Optional. Sends storefront and order event data to Adobe Experience Platform for personalization. Applies to Adobe Commerce on cloud infrastructure or on-premises, not Adobe Commerce as a Cloud Service. | <Link href="https://experienceleague.adobe.com/en/docs/commerce/data-connection/overview" text="Data Connection page" /> |

> | <Term>Storefront Compatibility package</Term> | Required for drop-ins. Manual install on Commerce PaaS or on-premises with Adobe Commerce Optimizer. Managed automatically on Adobe Commerce as a Cloud Service and Adobe Commerce Optimizer. | [Overview](/setup/configuration/storefront-compatibility/) · [Manual installation](/setup/configuration/storefront-compatibility/install/) |

Reviewer: ```suggestion
| <Term>Storefront Compatibility package</Term> | Required for drop-ins. Manual install on Commerce on Cloud or on-premises with Adobe Commerce Optimizer. Managed automatically on Adobe Commerce as a Cloud Service and Adobe Commerce Optimizer. | [Overview](/setup/configuration/storefront-compatibility/) · [Manual installation](/setup/configuration/storefront-compatibility/install/) |
```
